src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -655,6 +655,9 @@ await _sqlRetryService.ExecuteSql(
                                     sqlSearchOptions.SortHasMissingModifier = true;
                                 }
 
+                                _logger.LogInformation("Continuation token is {ContinuationTokenPresent}returned. {MaxSurrogateId}", continuationToken != null ? string.Empty : "not ", newContinuationId);
+                                _logger.LogInformation("Includes continuation token is {ContinuationTokenPresent}returned", includesContinuationTokenString != null ? string.Empty : "not ");
+
```

Comment:
nit: Add space i.e. {ContinuationTokenPresent}returned -> {ContinuationTokenPresent} returned

Reply:
I don't want the space there, it is in the string that replaces it. If I added it in the there then there would be two spaces when the continuation token is returned.

Reply:
Got it, thanks for clarifying!